.vscode/settings.json

```diff
@@ -40,7 +40,6 @@
         "editor.minimap.renderCharacters": false,
         "editor.minimap.showSlider": "always",
         "editor.suggest.localityBonus": true,
-        "editor.tabSize": 8,
```

Comment:
intentional?

Reply:
Oh.  Kinda ;)  I do this every time I check out PX4; didn't necessarily mean to include that in the PR though.  8-space tabs drive me insane.

Reply:
I'll revert the change :disappointed: